### PR TITLE
Update WeiHao Markdown.

### LINKED ISSUE #191: Markdown file final Wei Hao

### PR DESCRIPTION
**Pull request by chewwh09**
Update my contributions to the project in my markdown file.

Closes #191

### CODE REVIEW

**Review by JayChenYJ — APPROVED** [review]
LGTM

**Review by kumsssss — APPROVED** [review]
LGTM!

File: docs/team/chewwh09.md [hunk 1]
```diff
@@ -21,17 +21,25 @@ Given below are my contributions to the project.
...
+        * Edit initial `List` user guide (Pull requests [\#73](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/73))
+        * Add `reminder` feature (Pull requests [\#98](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/98))
+        * Update `reminder` feature (Pull requests [\#174](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/174))
+        * Modify and format User Guide (Pull requests [\#174](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/174))
     * Developer Guide:
-        * Added glossary. (Pull requests [\#42](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/42))
-        * Added use cases. (Pull requests [\#44](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/44))
+        * Add glossary. (Pull requests [\#42](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/42))
+        * Add use cases. (Pull requests [\#44](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/44))
+        * Add more use cases for `reminder` Feature. (Pull requests [\#98](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/98))
```
Comment (Md-Fazil) [inline docs/team/chewwh09.md:42]: I'd suggest including the link of all the PRS that you have reviewed and given input on.

**Review by Md-Fazil — COMMENTED** [review]
Some suggestions to consider. Otherwise, LGTM!

File: docs/team/chewwh09.md (hunk 1 above)
Comment (chewwh09) [reply docs/team/chewwh09.md:42]: All right noted!

**Pull request merged**